### ISSUE TITLE
You can wake yourself up with the noise if you're playing an instrument when you're falling asleep.

### ISSUE THREAD

**Issue by MrHrulgin** [CleverRaven/Cataclysm-DDA#29281]
**Describe the bug**  
You can wake yourself up with the noise if you're playing an instrument when you're falling asleep.

**To Reproduce**  
1. Get an instrument ( I specifically had this happen with a saxophone I was wearing)
2. Play the instrument
3. Without stopping, try to sleep.
4. Sometimes you'll wake up to the notice: "Something is making noise." 

**Expected behavior**  
I would expect that trying to sleep would stop you playing the instrument, as opposed to sometime around when you fall asleep.

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version: 0.D-1102-g7c3a038
 - Graphics version: Tiles
 - Mods loaded: so many mods

**Comment by zombiethoughts** [comment]
I've noticed this too since I always equip and play an instrument before going to bed to increase my morale.

But in my head I figured you're basically passing out while playing the instrument. So without the instrument put away properly, it's possible you fell asleep with the mouthpiece in your mouth/arm over the strings of a banjo and made a noise to wake yourself up.

Just my strange logic to explain why I was hearing noises while falling asleep.
